src/prompt.c

```diff
@@ -574,6 +574,11 @@ prompt_menu(const char *prompt, const struct menu_item *items, int *selected)
 			break;
 
 		default:
+			if (key.modifiers.control && (key.data.bytes[0] == 'C')) {
```

Comment:
But we are reading `data.bytes` two lines later at `if (items[i].hotkey == key.data.bytes[0]) {`.  Though perhaps the case should be `OK` rather than `default`.

I can't follow the logic behind the `multibytes` flag.  It is always set for control characters from `get_input()`, which is odd.  And the second condition here can never be true:

* https://github.com/jonas/tig/blob/e709657252f9fb09ff97c9cc6e2a159fc370ab27/src/display.c#L666

Was it originally meant to be `0x7F`? That would be logical as the alternative to `multibytes`.